test/SlowTests/Issues/RavenDB-23100.cs

```diff
@@ -0,0 +1,878 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Documents.Indexes.Counters;
+using Raven.Client.Documents.Indexes.TimeSeries;
+using Raven.Client.Documents.Operations.Backups;
+using Raven.Client.Documents.Session;
+using Raven.Server.ServerWide.Context;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_23100 : RavenTestBase
+    {
+        public RavenDB_23100(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        private const string EmployeeId1 = "Employ&es/1-A";
+        private const string EmployeeId2 = "EmPlo#ees/2-A";
+        private const string CompanyId = "COMan!es/1-A";
```

Comment:
Can you please use IDs resulting in having escape positions? (as we talked only for documents)

Reply:
done